src/DssProxyActionsCropper.sol

```diff
@@ -150,28 +148,21 @@ contract DssProxyActionsCharter is Common {
         address u,
         bytes32 ilk,
         uint256 wad
-    )  internal returns (int256 dart) {
+    ) internal returns (int256 dart) {
         // Updates stability fee rate
         uint256 rate = JugLike(jug).drip(ilk);
 
         // Gets DAI balance of the urn in the vat
         uint256 dai = VatLike(vat).dai(u);
 
-        // If there was already enough DAI in the vat balance,
-        //    just exits it without adding more debt
+        // If there was already enough DAI in the vat balance, just exits it without adding more debt
         uint256 rad = _mul(wad, RAY);
         if (dai < rad) {
-            uint256 netToDraw = rad - dai; // dai < rad
-
-            uint256 nib = (CharterLike(charter).gate(ilk) == 1) ?
-                CharterLike(charter).nib(ilk, u) :
-                CharterLike(charter).Nib(ilk);
-
+            uint256 toDraw = rad - dai; // dai < rad
             // Calculates the needed dart so together with the existing dai in the vat is enough to exit wad amount of DAI tokens
-            dart = _toInt256(_mul(netToDraw, WAD) / _sub(_mul(rate, WAD), _mul(rate, nib))); // wad
-            uint256 dtab = _mul(uint256(dart), rate);
-            // This is needed due lack of precision, it might need to sum an extra dart wei
-            dart = _sub(dtab, _mul(dtab, nib) / WAD) < netToDraw ? dart + 1 : dart;
+            dart = _toInt256(toDraw / rate);
+            // This is needed due lack of precision. It might need to sum an extra dart wei (for the given DAI wad amount)
+            dart = _mul(uint256(dart), rate) < toDraw ? dart + 1 : dart;
```

Comment:
Can use a simplified+optimized operation to do this in one line:

```
dart = _toInt256(1 + (toDraw - 1) / rate);
```

The subtraction is safe b/c we know `toDraw > 0` because `dai < rad`.

Reply:
addressed in [36bd557](https://github.com/makerdao/dss-crop-join/pull/26/commits/36bd557a2f0fc29b9b088531fcef2e5bd58edf86).

I ended up using _divup() which adds the function call and the `x != 0 ?` check which could have been avoided in our case. Think these are worth the benefit of using the standard function we (and others) use elsewhere and avoiding mistakes of other ppl porting the code.